signer/src/transaction_coordinator.rs

```diff
@@ -400,14 +402,28 @@ where
             tracing::debug!("no requests to handle, exiting");
             return Ok(());
         };
-
         tracing::debug!(
             num_deposits = %pending_requests.deposits.len(),
             num_withdrawals = pending_requests.withdrawals.len(),
             "fetched requests"
         );
         // Construct the transaction package and store it in the database.
         let transaction_package = pending_requests.construct_transactions()?;
+        // Get the requests from the transaction package because they have been split into
+        // multiple transactions.
+        let sbtc_requests = SbtcRequestsContextMessage {
+            requests: transaction_package
+                .iter()
+                .map(|tx| SbtcRequestsContext::from(&tx.requests))
+                .collect(),
+            fee_rate: pending_requests.signer_state.fee_rate,
+            last_fees: pending_requests.signer_state.last_fees.map(Into::into),
+        };
+
+        // Share the list of requests with the signers.
+        self.send_message(sbtc_requests, bitcoin_chain_tip).await?;
+        // Wait for the signers to receive and process the requests.
+        tokio::time::sleep(tokio::time::Duration::from_secs(1)).await;
```

Comment:
We might want to update the comment a little. We wait because if we proceed immediately and send the next message, there is a greater change that the other signers will receive subsequent messages before the `SbtcRequestsContextMessage` one.